themes/klasik/partials/content.php

```diff
@@ -70,9 +115,9 @@
 								<div class="img">
 									<?php if ($data['gambar']!=''): ?>
 										<?php if (is_file(LOKASI_FOTO_ARTIKEL."kecil_".$data['gambar'])): ?>
-											<img src="<?= AmbilFotoArtikel($data['gambar'],'kecil') ?>" alt="<?= $data["judul"] ?>"/>
+											<img src="<?= AmbilFotoArtikel($data['gambar'],'kecil') ?>" alt="<?= $data["judul"] ?>" width="300" style="float:left; margin:0 8px 4px 0;"/>
 										<?php else: ?>
-											<img src="<?= base_url('assets/images/404-image-not-found.jpg') ?>" alt="<?= $data["judul"] ?>" />
+											<img src="<?= base_url('assets/images/404-image-not-found.jpg') ?>" alt="<?= $data["judul"] ?>" width="300" style="float:left; margin:0 8px 4px 0;"/>
 										<?php endif;?>
```

Comment:
Stylenya bisa di defnisikan di awal agar tdk berulang dan inline.

Bisa pakai (is_file(LOKASI_FOTO_ARTIKEL."kecil_".$data['gambar'])) ? AmbilFotoArtikel($data['gambar'],'kecil') : base_url('assets/images/404-image-not-found.jpg');